### PR TITLE
implement add, sub, mul on posit

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: num/src/bigint.rs [hunk 1]
```diff
@@ -29,6 +29,12 @@ macro_rules! construct_bigint {
         pub struct $name([u64; $n_words]);
 
         impl $name {
+            #[inline]
+            /// Constructs an object from internal representation
+            pub const fn new(internal: [u64; $n_words]) -> $name {
```
Comment (dr-orlovsky) [inline num/src/bigint.rs:34]: It is recommended to use `from_inner` for the function names constructing newtype from the inner representation
```suggestion
            pub const fn from_inner(inner: [u64; $n_words]) -> $name {
```

File: num/src/bigint.rs [hunk 2]
```diff
@@ -1313,6 +1367,20 @@ macro_rules! construct_unsigned_bigint_methods {
     };
 }
 
+macro_rules! impl_into {
```
Comment (dr-orlovsky) [inline num/src/bigint.rs:1370]: why `impl_into` and not `impl_from`?

**Review by dr-orlovsky — COMMENTED** [review]
Two small questions/suggestions, otherwise LGTM

File: num/src/bigint.rs (hunk 2 above)
Comment (6293) [reply num/src/bigint.rs]: my bad naming choice. fixed

File: num/src/bigint.rs (hunk 1 above)
Comment (6293) [reply num/src/bigint.rs]: fixed

**Review by dr-orlovsky — APPROVED** [review]
ACK 188d7b3ecc820efc34f6e16c733e18072715bf41

**Pull request merged**